tests/e2e/constants/constants.go

```diff
@@ -1,9 +1,17 @@
 package constants
 
 const (
-	RunController = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./cloudcore > cloudcore.log 2>&1 &"
-	RunEdgecore   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edge/; sudo nohup ./edgecore > edgecore.log 2>&1 &"
-	RunEdgeSite   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edgesite/; sudo nohup ./edgesite > edgesite.log 2>&1 &"
+	RunCloudcore = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./cloudcore > cloudcore.log 2>&1 &"
+	RunEdgecore  = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edge/; sudo nohup ./edgecore > edgecore.log 2>&1 &"
+	RunEdgeSite  = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edgesite/; sudo nohup ./edgesite > edgesite.log 2>&1 &"
+
+	CheckCloudcore = "sudo pkill -0 cloudcore"
```

Comment:
why use pkill inside a check? you can record the pid when starting components and checkif the pid is still alive

Reply:
changed to `pgrep`